pkg/config/config.go

```diff
@@ -98,5 +102,9 @@ func FromFlags(progName string, args []string) (Config, string, error) {
 		flags.Usage()
 	}
 
+	if c.Version {
+		fmt.Println(version)
```

Comment:
Mhhh, this function should really just be parsing feature flags... Can we print the version directly in main if cfg.Version is set?